src/components/Login/store/reducer.js

```diff
@@ -0,0 +1,26 @@
+import {
+  USERNAME_INPUT_VALUE,
+  PASSWORD_INPUT_VALUE,
+  CLEAR_PASSWORD_VALUE,
+  FORM_SUBMIT
+} from './types';
+
+const initialState = {
+  password: '',
+  username: '',
+};
+
+export default function loginReducer(state = initialState, action) {
```

Comment:
If you are already using redux-actions lib (by the way why?) why you are not using handleActions function?